playerbot/PlayerbotFactory.cpp

```diff
@@ -913,12 +909,7 @@ void PlayerbotFactory::ResetQuests()
     ObjectMgr::QuestMap const& questTemplates = sObjectMgr.GetQuestTemplates();
     for (ObjectMgr::QuestMap::const_iterator i = questTemplates.begin(); i != questTemplates.end(); ++i)
     {
-#ifndef MANGOSBOT_TWO  
-        Quest const* quest = i->second;
-#else
         Quest const* quest = i->second.get();
-#endif;
```

Comment:
Not sure why this extra semicolon is here, it wasn't here before the preprocessor defines.

Reply:
Call it magic butter fingers